statsmodels/stats/stattools.py

```diff
@@ -76,27 +76,42 @@ def omni_normtest(resids, axis=0):
 
 
 def jarque_bera(resids, axis=0):
-    """
-    Calculate residual skewness, kurtosis, and do the JB test for normality
+    r"""
+    Calculates the Jarque-Bera test for normality
 
     Parameters
     -----------
-    resids : array-like
+    data : array-like
+        Data to test for normality
     axis : int, optional
-        Default is 0
+        Axis to use if data has more than 1 dimension. Default is 0
 
     Returns
     -------
-    JB, JBpv, skew, kurtosis
+    JB : float or array
+        The Jarque-Bera test statistic
+    JBpv : float or array
+        The pvalue of the test statistic
+    skew : float or array
+        Estimated skewness of the data
+    kurtosis : float or array
+        Estimated kurtosis of the data
 
-    JB = n/6*(S^2 + (K-3)^2/4)
+    Notes
+    -----
+    Each output returned has 1 dimension fewer than data
 
-    JBpv is the Chi^2 two-tail probability value
+    The Jarque-Bera test statistic tests the null that the data is normal
+    against an alternative that it it is not. It is based on two moments of
```

Comment:
I would add "distributed" after normal. AFAICS the entire docstring doesn't mention "distribution"